### DISCUSSION TITLE
is keymap ability coming?  / ability to save keymap loadouts?

### DISCUSSION THREAD

**Discussion by xoxo369** [flit/MidiKeys#23] [Q&A]
I just found this program after trying 4 other ones and I'm so grateful, this one works the best for me but I really wish I could remap the keys and if you could save keymaps this would also add the ability to save scales and that idea excites me a lot, hope to see updates in the near future and THANK YOU!!!! <3 been needing something like this and this one works the best for me :)

**Comment by bramst0ne** [comment]
just edit the keymaps.plist for now.
